hasjob/models/jobpost.py

```diff
@@ -3,18 +3,19 @@
 from collections import defaultdict
 from datetime import datetime, timedelta
 from werkzeug import cached_property
-from flask import url_for, escape, Markup
+from flask import g, url_for, escape, Markup
```

Comment:
What is `g` being used for? Importing it in a model is an indication that something is wrong.